teos/src/api/internal.rs

```diff
@@ -542,19 +582,23 @@ mod tests_private_api {
                     .unwrap();
             }
 
+            // Add that many appointments to the watcher.
             let locator = Locator::new(dispute_txid);
```

Comment:
This comment is unrelated.
Also let's move this `locator` definition to the top.